lib/deploy/activate.rb

```diff
@@ -87,16 +89,16 @@ def symlink_verbose(dest, link, force: false)
 
     def download_application_yml_from_s3
       LoginGov::Hostdata.s3(logger: logger, s3_client: s3_client).download_configs(
-        '/%<env>s/idp/v1/application.yml' => env_yaml_path,
+        '/%<env>s/idp/v1/application.yml' => @env_yaml_path,
```

Comment:
what about adding `attr_reader`s for `env_yaml_path` and `result_yaml_path`?

Reply:
quick note, not a huge deal, the thing I meant about the `attr_reader` is we could eliminate this line noise by using those methods here

Reply:
ohhhh, right 🤦🏼 

I realized the fakefs dependency could be removed, and so have a new pull request anyway that now includes those changes: https://github.com/18F/identity-idp/pull/4026